tests/test_self.py

```diff
@@ -74,10 +74,11 @@ def _configure_lc_ctype(lc_ctype: str) -> Iterator:
             os.environ[lc_ctype_env] = original_lctype
 
 
+# noinspection PyMissingConstructor
```

Comment:
For which tool is this?

Reply:
Pycharm :)

Reply:
Ah. Not sure we should add it. For gitignore we also leave out any IDE specific pragma's.

_There are also many things VS Code would like us to fix 😅_

Reply:
```suggestion
```
Fair enough !